chart/values.yaml

```diff
@@ -0,0 +1,66 @@
+# Default values for trow.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: containersol/trow
+  pullPolicy: Always
+
+trow:
+  domain: myregistry.mydomain.io
+  user: user
+  password: password
+  webhooks: false
```

Comment:
This controls the validating admission webhook, right? Maybe `validation` or `validatingWebhook` would be better. And lets add a comment to explain.

Reply:
OKay perfect

Reply:
and need to do this